kraken/kubernetes/client.py

```diff
@@ -14,10 +17,17 @@
 def initialize_clients(kubeconfig_path):
     global cli
     global batch_cli
+    global api_client
+    global dyn_client
+    global custom_object_client
     try:
         config.load_kube_config(kubeconfig_path)
         cli = client.CoreV1Api()
         batch_cli = client.BatchV1Api()
+        api_client = client.ApiClient()
+        custom_object_client = client.CustomObjectsApi()
```

Comment:
Since we are now setting a global custom object api can you use that object in the function _get_clusterversion_string_. It is double initializing it